autogen.sh

```diff
@@ -1,11 +1,41 @@
 #!/bin/sh
+# Run this to generate all the initial makefiles, etc.
+test -n "$srcdir" || srcdir=$(dirname "$0")
+test -n "$srcdir" || srcdir=.
 
-PKG_NAME="indicator-messages"
+olddir=$(pwd)
 
-which gnome-autogen.sh || {
-	echo "You need gnome-common from GNOME SVN"
-	exit 1
+cd $srcdir
+
+(test -f configure.ac) || {
+        echo "*** ERROR: Directory '$srcdir' does not look like the top-level project directory ***"
+        exit 1
 }
 
-USE_GNOME2_MACROS=1 \
-. gnome-autogen.sh
+# shellcheck disable=SC2016
+PKG_NAME=$(autoconf --trace 'AC_INIT:$1' configure.ac)
+
+if [ "$#" = 0 -a "x$NOCONFIGURE" = "x" ]; then
+        echo "*** WARNING: I am going to run 'configure' with no arguments." >&2
+        echo "*** If you wish to pass any to it, please specify them on the" >&2
+        echo "*** '$0' command line." >&2
+        echo "" >&2
+fi
+
+aclocal --install || exit 1
+intltoolize --force --copy --automake || exit 1
+gtkdocize --copy || exit 1
+autoreconf --verbose --force --install || exit 1
+
+cd "$olddir"
+if [ "$NOCONFIGURE" = "" ]; then
```

Comment:
You have to also update `debian/rules`; it currently uses `NO_CONFIGURE`.

Reply:
Good catch, fixed